README.md

```diff
@@ -2,3 +2,81 @@
 Parse Public Cloud image names that match the SUSE naming convention
 
 API implementation to parse image names that match the SUSE Public Cloud image naming convention.
+
+## Quick Start
+
+### Install from Source
+
+```bash
+  # Clone the repo
+  $ git clone https://github.com/SUSE-Enceladus/suse-cloud-image-name-parser
+  $ cd suse-cloud-image-name-parser 
+
+  # create the virtual environment.
+  $ virtualenv dev_venv --python=python3
+
+  # activate the virtual environment for interactive bash session
+  $ source dev_venv/bin/activate
+
+  # There is 2 ways you can install
+    1. Install latest sources in developer mode
+       $ pip install -e .[dev]
+
+    2. Install latest version normally
+       $ pip install .[dev]
+```
+
+## Running Unit Tests
+
+```bash
+  # Clone the repo
+  $ git clone https://github.com/SUSE-Enceladus/suse-cloud-image-name-parser
+  $ cd suse-cloud-image-name-parser
+
+  # create the Python 3.6  virtual environment.
```

Comment:
This comment is incorrect. The command that follows will setup a Python 3 venv based on whatever version of the Python 3 interpreter the `python3` command on the PATH is linked to or whatever Python 3 version executable is built as the `python3` command on the system.

`virtualenv test_venv --python=python3`

does not guarantee a Python 3.6 venv setup as the command states.